app/models/agents/webhook_agent.rb

```diff
@@ -100,8 +100,14 @@ def receive_web_request(request)
           return ["Not Authorized", 401]
         end
 
-        JSON.parse(response.body)['success'] or
-          return ["Not Authorized", 401]
+        body = JSON.parse(response.body)
+        if body['score'].presence
```

Comment:
We always try to not change or break current configurations of Agents, so this branch must be "guarded" behind a new Agent configuration option. Something like this could work:
```ruby
         if interpolated['score_threshold'].present? && body['score'].present?
````